docs/paper/admin/reference/configuration/spigot-configuration.mdx

```diff
@@ -0,0 +1,563 @@
+        description: 'The entity activation range for villagers.'
+      villagers-active-for-panic:
+        default: 'true'
+        description: 'Whether to activate villagers when they want to panic.'
+      villagers-work-immunity-after:
+        default: '100'
+        description: 'The time in ticks a villager has to be inactive and working to be woken up.'
+      villagers-work-immunity-for:
+        default: '20'
+        description: 'How long a villager should be woken up for, in ticks, after villagers-work-immunity-after.'
+      wake-up-inactive:
+        animals-every:
+          default: '1200'
+          description: 'How often an inactive animal outside of range will be woken up, in ticks.
+            This is a minimum due to being limited to animals-max-per-tick.'
+        animals-for:
+          default: '100'
+          description: 'How long to wake an inactive animal up for, in ticks.'
+        animals-max-per-tick:
+          default: '4'
+          description: 'A limit of how many inactive animals can be woken up on the same tick.'
+        flying-monsters-every:
+          default: '200'
+          description: 'How often an inactive flying monster outside of range will be woken up, in ticks.
+            This is a minimum due to being limited to flying-monsters-max-per-tick.'
+        flying-monsters-for:
+          default: '100'
+          description: 'How long to wake an inactive flying monster up for, in ticks.'
+        flying-monsters-max-per-tick:
+          default: '8'
+          description: 'A limit of how many inactive flying monsters can be woken up on the same tick.'
+        monsters-every:
+          default: '400'
+          description: 'How often an inactive monster outside of range will be woken up, in ticks.
+            This is a minimum due to being limited to monsters-max-per-tick.'
+        monsters-for:
+          default: '100'
+          description: 'How long to wake an inactive monster up for, in ticks.'
+        monsters-max-per-tick:
+          default: '8'
+          description: 'A limit of how many inactive monsters can be woken up on the same tick.'
+        villagers-every:
+          default: '600'
+          description: 'How often an inactive villager outside of range will be woken up, in ticks.
+            This is a minimum due to being limited to villagers-max-per-tick'
+        villagers-for:
+          default: '100'
+          description: 'How long to wake an inactive villager up for, in ticks.'
+        villagers-max-per-tick:
+          default: '4'
+          description: 'A limit of how many inactive villagers can be woken up on the same tick.'
+      water:
+        default: '16'
+        description: 'The entity activation range for water mobs.'
+    entity-tracking-range:
+      animals:
+        default: '48'
+        description: 'Controls how far in blocks animals are tracked (sent to) the player.
+          This is scaled by entityBroadcastRangePercentage.'
```

Comment:
This would be convenient if all other settings reference was clickable to their own page. Applies for others

Reply:
done